xedocs/xedocs.py

```diff
@@ -127,6 +127,20 @@ def xedocs_protocol(name, version="ONLINE", sort=None, attr=None, **kwargs):
             return docs[0]
 
         return docs
+    
+    @URLConfig.register('docs-to-array')
```

Comment:
can be used with any list, not just a list of docs
```suggestion
    @URLConfig.register('list-to-array')
```